src/pages/app/App.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  BrowserContextProvider,
+  TranslationContextProvider,
+} from '@/pages/shared/lib/context';
+import browser from 'webextension-polyfill';
+import {
+  createMemoryRouter,
+  RouterProvider,
+  type RouteObject,
+} from 'react-router-dom';
+import { MessageContextProvider, WaitForStateLoad } from './lib/context';
+
+export const ROUTES = {
+  HOME: '/',
```

Comment:
What is the home route being used for?

Reply:
Unused for now, just there for router default page. I think Home will correspond to a Settings layout in future (/home/setting1, /home/setting2).
Logic for showing what route we want to render will come in follow-up PRs as well implement them. For example, if connected once already, showing post install doesn't make sense again - so Home would probably just show a message like "use popup to control extension" until Home can have some actual content.